src/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>To Do List</title>
+    <link rel="stylesheet" href="styles.css">
+    
+</head>
+<body>    
+    <div class="container">
+        <div class="header">
+            <h3 class="today">Today's To Do<i class="fa-light fa-arrows-rotate"></i></h3>
+            <hr>
+            <div class="row"> 
+                <input type="text" id="type-todo" placeholder="Add to your list....">
+                <div id="error-message"></div>          
+                
+            </div>
+            <hr>
+            
+         <div id="todo-list"></div> 
+         <div class="submit-btn">      
+             <input type="submit" name="submit" id="submit" value="Clear all Completed">       
+         </div>
+</div>     
+    <script src="../dist/index.bundle.js"></script>    
+</body>
```

Comment:
To follow JavaScript Best Practice, you may keep all the JavaScript code in JS files, not in the **script**  tag